src/shared/label-util.c

```diff
@@ -99,6 +99,24 @@ int mknod_label(const char *pathname, mode_t mode, dev_t dev) {
         return mac_smack_fix(pathname, 0);
 }
 
+int mknodat_label(int dirfd, const char *pathname, mode_t mode, dev_t dev) {
```

Comment:
Please make mknod_label a static inline wrapper of this function

Reply:
done